scripts/verifyOldTs.mjs

```diff
@@ -144,7 +144,10 @@ function typeTests() {
 
     verifyInstalledTypescript();
 
-    execa.sync('yarn', ['test-types'], {cwd, stdio: 'inherit'});
+    execa.sync('yarn', ['test-ts', '--selectProjects', 'type-tests'], {
```

Comment:
Overlooked it before. Current script runs TSTyche as well. This is just waste of time, because `yarn tstyche --target 5.0,current` already tested on TypeScript 5.0.